superbench/benchmarks/micro_benchmarks/rocm_memory_bw_performance.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
One quick question: I find this code is almost the same as cuda_memory_bw_performance, but the parameters are different, why? Shall we need to merge it directly?